.scripts/serve.py

```diff
@@ -69,12 +76,16 @@ def do_GET(self):
             self._send_json(execute_function(Functions, data))
         elif self.path == "/functions":
             self._send_json(get_all_metadata(Functions))
-        elif self.path == "/functions.html":
+        elif self.path.startswith("/functions.html"):

```

Comment:
You're not using any parameters on this URL, so we can go back to `==`?

Reply:
Excel WAC will append some querystring to the URL.